src/mupen64plus-video-gliden64.mk

```diff
@@ -128,13 +128,15 @@ ifeq ($(TARGET_ARCH_ABI), armeabi-v7a)
     # Use for ARM7a:
     MY_LOCAL_SRC_FILES += $(SRCDIR)/3DMathNeon.cpp.neon
     MY_LOCAL_SRC_FILES += $(SRCDIR)/gSPNeon.cpp.neon
+    MY_LOCAL_SRC_FILES += $(SRCDIR)/RSP_LoadMatrixNeon.cpp.neon
```

Comment:
I don't know. But 3DMathNeon.cpp.neon and gSPNeon.cpp.neon should be also wrong then. As i asked fzurita these lines look suspicious.